build.sbt

```diff
@@ -185,18 +186,19 @@ lazy val `kyo-tapir` =
         )
 
 lazy val `kyo-test` =
-  crossProject(JVMPlatform, JSPlatform)
-    .withoutSuffixFor(JVMPlatform)
-    .crossType(CrossType.Full)
-    .in(file("kyo-test"))
-    .dependsOn(`kyo-core` % "test->test;compile->compile")
```

Comment:
The test dependency to `kyo-core` was the cause of the issue with `@EnableReflectiveInstantiation`.